TASK/SetPedCanPlayAmbientIdles.md

```diff
@@ -9,15 +9,16 @@ aliases: ["0x8FD89A6240813FD0"]
 void SET_PED_CAN_PLAY_AMBIENT_IDLES(Ped ped, BOOL p1, BOOL p2);
 ```
 
-```
-Appears only in fm_mission_controller and used only 3 times.  
-ped was always PLAYER_PED_ID()  
-p1 was always true  
-p2 was always true  
+Disables the idle animation that starts playing when the player ped has been standing still for a few seconds. Has to be called every frame. Does not work if `p1` and/or `p2` are `false`, both parameters have to be `true`. Native does not sync over network (you have to call it on each client for all active player peds).
+
+```lua
+while true do
+    SetPedCanPlayAmbientIdles(PlayerPedId(), true, true)
+    Citizen.Wait(0)
+end
 ```
 
 ## Parameters
 * **ped**: 
 * **p1**: 
 * **p2**: 
```

Comment:
* **ped**: The ped index.
* **blockidleclips**: If true, blocks future idle clips from playing.
* **removeclip**: If true, will remove any current clip playing (will also block).


This is the missing params :D

Reply:
thank you! I'll update my PR

Reply:
merged with master